Client/Frontend/Settings/Rewards Internals/RewardsInternalsContributionListController.swift

```diff
@@ -48,6 +48,8 @@ extension ContributionStep {
             return s.contributionsStepExternalTransaction
         case .stepCreds:
             return s.contributionsStepCreds
+        case .stepRetryCount:
+            fallthrough
```

Comment:
interesting, swiftLint likes `fallthrough`? 
I thought it's generally a bad pattern, at least when I worked with Java

Reply:
Yeah it's fine with `fallthrough`, I don't believe this screen is visible anymore to the user so I just silenced the warning by adding the case (even though before it was hitting the `@unknown default` clause at runtime anyways)